app/views/room.html

```diff
@@ -16,7 +16,7 @@ <h4 ng-bind="room.name"></h4>
                 <input type="text" ng-model="chatMessage.message" placeholder="Type a message" maxlength="255" />
                 <input type="submit" value="Send" class="highlight" />
             </form>
-            <div ng-hide="loggedIn">  In order to chat you need to be logged in.</div>
+            <div ng-hide="loggedIn" class="notloggedIn">You must be logged in to chat! You can log in with your SounCloud account in the top right.</div>
```

Comment:
I was thinking, do we want make clear here that it's soundcloud then people think I don't have that.
While clicking on the button also displays google and facebook login (for soundcloud).